doc/src/manual/methods.md

```diff
@@ -698,11 +698,14 @@ While this works for declared types, it fails for types without
 supertypes:
 
 ```julia-repl
-julia> eltype_wrong(Union{AbstractArray{Int}, AbstractArray{Float64}})
-ERROR: MethodError: no method matching supertype(::Type{Union{AbstractArray{Float64,N} where N, AbstractArray{Int64,N} where N}})
-Closest candidates are:
-  supertype(::DataType) at operators.jl:43
-  supertype(::UnionAll) at operators.jl:48
+julia> eltype_wrong(Union{Vector{Int}, Matrix{Int}})
+ERROR: MethodError: no method matching supertype(::Type{VecOrMat{Int64}})
+
+Closest candidates are:  
+  supertype(::UnionAll)  
+   @ Base operators.jl:44
+  supertype(::DataType)  
```

Comment:
```suggestion
Closest candidates are: 
  supertype(::UnionAll)
   @ Base operators.jl:44
  supertype(::DataType)
```

Reply:
There's still some trailing whitespace left after the suggested change :smile:

Reply:
> `Closest candidates are: `